{{cookiecutter.project_name}}/monitoring/README.md

```diff
@@ -0,0 +1,63 @@
+# Setup Model Monitoring
+
+This directory contains setup scripts intended to generate notebook that can be uploaded to databricks workspace and execute
+to set up model monitoring.
+
+## Prerequisites
+
+### Unity Catalog
+The inference table to be monitored must exist in Unity Catalog.
+If Unity Catalog is enabled for your workspace but the table exists in metastore, please run notebook code 
+or considering create workflow to copy data to Unit Catalog.
+
+For example, the following code 
+ - load data from metastore example_project_predictions 
+ - write data to Unity Catalog my_unity_catalog.my_schema.my_inference_table
+ - convert prediction column to double type
+ - convert inference_time_stamp to timestamp type
+ - convert tpep_pickup_datetime to timestamp type
+ - convert tpep_dropoff_datetime to timestamp type
+```
+df = spark.table("hive_metastore.default.example_project_predictions") \
+        .withColumn("prediction",col("prediction").cast("double")) \
+        .withColumn("inference_time_stamp",to_timestamp("inference_time_stamp")) \
+        .withColumn("tpep_pickup_datetime",to_timestamp("tpep_pickup_datetime")) \
+        .withColumn("tpep_dropoff_datetime",to_timestamp("tpep_dropoff_datetime"))
+df.write.saveAsTable(
+    name = "my_unity_catalog.my_schema.my_inference_table"
+)
+```
+
+### Required columns
+
+The inference table is required to contain the following columns
+ - a column that contains the prediction results. For regression problems the type must be number type.
+ - a column with timestamp type that contains the time that the inference happens
+ - a column with string type that represents the model version used to predict
+
+
+## Usage
+
+### 1. Run the scripts
+From the monitoring directory, run:
+
+```
+python bootstrap.py
+```
+
+A notebook `output/setup_model_monitoring.py` will be generated in the monitoring directory.
+
+### 2. Update optional fields as necessary
+
+The notebook `output/setup_model_monitoring.py` has required fields already filled in. Read the instructions 
+in the notebook to update optional fields or add custom metrics.
+
+### 3. Set up monitoring
+
+Upload the generated notebook file `output/setup_model_monitoring.py` to workspace and run it to generate 
```

Comment:
Is this a one time thing? Curious if it's worth adding scaffolding in Stacks for it (e.g., terraform). Probably not worth it right?